swift_browser_ui_frontend/src/components/ShareModal.vue

```diff
@@ -53,17 +50,29 @@
             </c-link>
           </c-flex>
         </c-row>
-        <ul
+        <div
           v-show="openShareGuide"
-          class="guide-content"
+          class="content guide-content"
         >
-          <li>
+          <p>
+            {{ $t("message.share.share_guide_intro") }}
+          </p>
+          <p>
             {{ $t("message.share.share_guide_step1") }}
-          </li>
-          <li>
+          </p>
+          <p>
             {{ $t("message.share.share_guide_step2") }}
-          </li>
-        </ul>
+          </p>
+          <ul>
+            <li
+              v-for="(item, i) in
+                $t('message.share.share_guide_step2_list')"
```

Comment:
```suggestion
                $tm('message.share.share_guide_step2_list')"
```
`vue-i18n-next` no longer returns strings with `$t`, `$tm` has been provided for fetching objects/arrays. Currently the rendering is somewhat broken.
![Screenshot from 2023-03-29 12-46-41](https://user-images.githubusercontent.com/24293337/228524766-f3ba732d-2bd8-43a8-bc4d-912911ce7b0e.png)